webb_ai/assets/dashboards/webbai_rca.json

```diff
@@ -0,0 +1,114 @@
+{
+  "title": "Webb.ai RCAs",
```

Comment:
This should not link out to your own dashboard. This should be out of the box dashboard for users to look and visualize data

Reply:
Please use the template I sent over previously